consensus/polybft/blockchain_wrapper.go

```diff
@@ -108,6 +110,9 @@ func (p *blockchainWrapper) ProcessBlock(parent *types.Header, block *types.Bloc
 
 	_, root := transition.Commit()
 
+	metrics.SetGauge([]string{consensusMetricsPrefix, "block_execution_time"},
+		float32(time.Now().UTC().Sub(start).Seconds()))
```

Comment:
Move it to the `consensus_metrics.go` (`updateBlockMetrics` function).

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1564/commits/de7a671831ce441316a1fd172270314a90ad0c70